commit 225de11e31c1cecd04839b859a0b8f81d490a50b
Author: Steve French <smfrench@gmail.com>
Date:   Tue Jan 3 23:08:24 2012 -0600

    [CIFS] default ntlmv2 for cifs mount delayed to 3.3
    
    Turned out the ntlmv2 (default security authentication)
    upgrade was harder to test than expected, and we ran
    out of time to test against Apple and a few other servers
    that we wanted to.  Delay upgrade of default security
    from ntlm to ntlmv2 (on mount) to 3.3.  Still works
    fine to specify it explicitly via "sec=ntlmv2" so this
    should be fine.
    
    Acked-by: Jeff Layton <jlayton@samba.org>
    Signed-off-by: Steve French <smfrench@gmail.com>

diff --git a/fs/cifs/connect.c b/fs/cifs/connect.c
index 27c4f2551711..f3670cf72587 100644
--- a/fs/cifs/connect.c
+++ b/fs/cifs/connect.c
@@ -2122,7 +2122,7 @@ cifs_get_smb_ses(struct TCP_Server_Info *server, struct smb_vol *volume_info)
 		warned_on_ntlm = true;
 		cERROR(1, "default security mechanism requested.  The default "
 			"security mechanism will be upgraded from ntlm to "
-			"ntlmv2 in kernel release 3.2");
+			"ntlmv2 in kernel release 3.3");
 	}
 	ses->overrideSecFlg = volume_info->secFlg;